docs/learn/dapp-developer/readData.md

```diff
@@ -94,12 +84,13 @@ async function fetchProfile(address) {
     const profile = new ERC725(erc725schema, address, provider, config);
     return await profile.fetchData();
   } catch (error) {
+    console.log(error);
     return console.log('This is not an ERC725 Contract');
   }
 }
 
 // Debug
-fetchProfile(SAMPLE_PROFILE_ADDRESS).then((profileData) =>
+fetchProfile(0x6979474ecb890a8efe37dab2b9b66b32127237f7).then((profileData) =>
```

Comment:
This is duplicated content if we touch on the base functionality. It's calling the above function and has the `JSON.stringify` in it. As Fabian stated, we should remove functions and string-modifications (just keep the core). The profile address can directly go into the line of my suggestion above.